llvm/lib/Target/BPF/BPFInstrInfo.td

```diff
@@ -790,7 +790,7 @@ let Predicates = [BPFNoALU32] in {
 class ATOMIC_NOFETCH<BPFWidthModifer SizeOp, string OpType, RegisterClass RegTp,
                      BPFArithOp Opc, string Opstr>
     : TYPE_LD_ST<BPF_ATOMIC.Value, SizeOp.Value,
-                 (outs GPR:$dst),
+                 (RegTp:GPR:$dst),
```

Comment:
Should this be `(outs RegTp:$dst)`?

Reply:
It should. Somehow I made a mistake in the change. Fixed it now.